cmd/runtimetest/main.go

```diff
@@ -271,6 +271,62 @@ func validateDefaultFS(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateLinuxDevices(spec *rspec.Spec) error {
+	logrus.Debugf("validating linux devices")
+
+	for _, device := range spec.Linux.Devices {
+		fi, err := os.Stat(device.Path)
+		if err != nil {
+			return err
+		}
```

Comment:
On Wed, Sep 14, 2016 at 01:39:06AM -0700, Ma Shimiao wrote:
> As you know, almost all validation error out, if config did not
> match container.  Though we added a caveat in help message, the
> runtimetest still could not finish whole testcase.

We can address that when we switch from the current “lots of tests run
from within one container” to a test harness that lets us run “lots of
little one-test containers”.  And I don't think it's worth reading the
config to detect these corner cases when we can say “this doesn't
cover all the corner cases” and then only call runtimetest in
situations where it works ;).